### PR TITLE
Feature/Add 2 user fields and update pytest, integration test

### PR DESCRIPTION
**Pull request by YasinFatullayev**
Add gender&birthday fields to user.
Update unittest with pytest and integration test.

### CODE REVIEW

File: integration-testing/schema/mutations.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ const gql = require('graphql-tag')
...
       userId
       username
       fullName
       email
       phoneNumber
+      birthday
+      gender
       signedUpAt
     }
   }
```
Comment (mfogel) [inline integration-testing/schema/mutations.js:17]: I should have mentioned this earlier - but we don't need to change `Mutation.createCognitoOnlyUser()`. We only need to add the `birthday` and `gender` arguments to `Mutation.setUserDetails()` for now.

File: integration-testing/tests/users/properties.test.js [hunk 2]
```diff
@@ -74,6 +75,23 @@ describe('Read and write properties our our own profile', () => {
     expect(resp.data.user.bio).toBeNull()
     expect(resp.data.user.fullName).toBeNull()
   })
+
+  test('Birthday and Gender', async () => {
+    const birthday = moment('1900-01-01').format('YYYY-MM-DD')
```
Comment (mfogel) [inline integration-testing/tests/users/properties.test.js:80]: I would either make this

`const birthday = '1900-01-01'`

or 

`const birthday = moment().format('YYYY-MM-DD')`

File: integration-testing/tests/users/properties.test.js [hunk 3]
```diff
@@ -74,6 +75,23 @@ describe('Read and write properties our our own profile', () => {
...
+
+    let resp = await client.query({query: queries.user, variables: {userId}})
+
+    // Set values to current User
+    resp = await client.mutate({mutation: mutations.setUserDetails, variables: {birthday, gender: 'MALE'}})
+    expect(resp.data.setUserDetails.birthday).toBe(birthday)
+    expect(resp.data.setUserDetails.gender).toBe('MALE')
+
+    // Set another values to current User
+    resp = await client.mutate({mutation: mutations.setUserDetails, variables: {birthday, gender: 'FEMALE'}})
```
Comment (mfogel) [inline integration-testing/tests/users/properties.test.js:91]: can we also test changing the birthday in this call as well?

File: real-main/app/models/user/enums.py [hunk 4]
```diff
@@ -20,3 +20,10 @@ class UserSubscriptionLevel:
 
     _ALL = (BASIC, DIAMOND)
     _PAID = (DIAMOND,)
+
+
+class UserGender:
+    MALE = 'MALE'
+    FEMALE = 'FEMALE'
+
+    _ALL = (MALE, FEMALE)
```
Comment (mfogel) [inline real-main/app/models/user/enums.py:29]: file missing trailing line ending

File: real-main/app_tests/models/user/test_dynamo.py [hunk 5]
```diff
@@ -495,6 +515,8 @@ def test_set_last_client(user_dynamo):
         ['increment_post_viewed_by_count', 'decrement_post_viewed_by_count', 'postViewedByCount'],
     ],
 )
+
+
```
Comment (mfogel) [inline real-main/app_tests/models/user/test_dynamo.py:519]: extra whitespace

**Comment by mfogel** [comment]
Oh - one other thing @YasinFatullayev . Please add the two new dynamo fields to the [SCHEMA.md ](https://github.com/real-social-media/backend/blob/master/SCHEMA.md) document. Let me know if you have questions, thanks

**Comment by YasinFatullayev** [comment]
@mfogel , I've fixed as your opinion. Please check.

File: integration-testing/tests/users/properties.test.js [hunk 6]
```diff
@@ -74,6 +75,27 @@ describe('Read and write properties our our own profile', () => {
...
+    resp = await client.mutate({mutation: mutations.setUserDetails, variables: {birthday, gender: 'MALE'}})
+    expect(resp.data.setUserDetails.birthday).toBe(birthday)
+    expect(resp.data.setUserDetails.gender).toBe('MALE')
+
+    // Set another values to current User
+    resp = await client.mutate({
+      mutation: mutations.setUserDetails,
+      variables: {anotherBirthday, gender: 'FEMALE'},
+    })
+    expect(resp.data.setUserDetails.birthday).toBe(birthday)
```
Comment (mfogel) [inline integration-testing/tests/users/properties.test.js:96]: I think this should be `expect(resp.data.setUserDetails.birthday).toBe(anotherBirthday)`?

File: real-main/app/handlers/appsync/handlers.py [hunk 7]
```diff
@@ -254,6 +258,13 @@ def set_user_details(caller_user, arguments, **kwargs):
     if privacy_status is not None:
         caller_user.set_privacy_status(privacy_status)
 
+    # Set Gender
+    if gender is not None:
+        try:
+            caller_user.set_gender(gender)
+        except UserException as err:
+            raise ClientException(str(err))
+
```
Comment (mfogel) [inline real-main/app/handlers/appsync/handlers.py:267]: Would it work to pass `gender` as another kwarg to `caller_user.update_details()`, just like `birthday` ?

That might let us eliminate `User.set_gender()` and `UserDynamo.set_user_gender()`

**Pull request merged**